docs/getting-started.mdx

```diff
@@ -99,6 +99,18 @@ Head to the [Version Policy page](/releases) to know which versions of TypeScrip
 and make sure that the [`@types/node`](https://www.npmjs.com/package/@types/node) package corresponding to your Node.js version is installed
 in your project.
 
+:::info
+
+Sequelize makes use of the the [`exports`](https://nodejs.org/api/packages.html#exports) `package.json` field.
+You may need to update your `tsconfig.json` configuration to make sure TypeScript's module resolution supports it.
+
+At time of writing (TS 5.2), this can be done in two ways:
```

Comment:
We can update this to 5.4, but I don't think the settings have changed

Reply:
Done